include/deal.II/integrators/advection.h

```diff
@@ -256,7 +340,128 @@ namespace LocalIntegrators
             }
         }
     }
+    /**
+     * Scalar: Residual for upwind flux at the boundary for weak advection operator. This is the
+     * value of the trial function at the outflow boundary and zero else:
+     * @f[
+     * a_{ij} = \int_{\partial\Omega}
+     * [\mathbf w\cdot\mathbf n]_+
+     * (u_i-g) v_j \, ds
+     * @f]
```

Comment:
Sorry for my confusion, but don't I just want to the boundary condition information on the inflow part of the domain for the domain in an upwind flux? In other words, should it be just the case `nv<0` what I want to add below? Or am I misunderstanding something here?